statsmodels/tsa/innovations/_arma_innovations.pyx.in

```diff
@@ -0,0 +1,734 @@
+#cython: boundscheck=False
+#cython: wraparound=False
+#cython: cdivision=False
+"""
+Innovations algorithm
+
+Author: Chad Fulton
+License: Simplified-BSD
+"""
+
+{{py:
+
+TYPES = {
+    "s": ("np.float32_t", "np.float32", "np.NPY_FLOAT32"),
+    "d": ("np.float64_t", "float", "np.NPY_FLOAT64"),
+    "c": ("np.complex64_t", "np.complex64", "np.NPY_COMPLEX64"),
+    "z": ("np.complex128_t", "complex", "np.NPY_COMPLEX128"),
+}
+
+}}
+
+# Typical imports
+import numpy as np
+import warnings
+from statsmodels.tsa import arima_process
+from statsmodels.src.math cimport *
+cimport numpy as np
```

Comment:
Can you make this `cimport numpy as cnp`?  I find it helps to keep the python-numpy and cython-numpy namespaces explicitly distinct

Reply:
Good idea